django/db/models/query.py

```diff
@@ -719,8 +723,9 @@ def _values(self, *fields):
 
         return clone
 
-    def values(self, *fields):
-        clone = self._values(*fields)
+    def values(self, *fields, **expressions):
+        fields += tuple(expressions)
```

Comment:
We need to do this here instead of in `_values` because `values_list` needs to manipulate `fields` too.